kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/extensions/KroxyliciousExtension.java

```diff
@@ -42,15 +41,15 @@ public KroxyliciousExtension() {
 
     @Override
     public boolean supportsParameter(ParameterContext parameterContext, ExtensionContext extensionContext) throws ParameterResolutionException {
-        return !parameterContext.getParameter().getType().isAssignableFrom(TestInfo.class);
+        return parameterContext.getParameter().getType().isAssignableFrom(String.class);
```

Comment:
I wonder if it would be clearer if the parameter type was io.fabric8.kubernetes.api.model.Namespace?

Reply:
I think this is adding complexity to the test cases that are not needed. This `Namespace` is not just a simple `String` that is what we need for our test cases. This is a complex model that contains its respective metadata, spec... etc etc properties from a kubernetes configuration.

I don't like either the idea to set here `String.class` but the other option is to negate those who are not allowed like I set before `TestInfo.class`  and `TestKmsFacade.class`. 

I didn't found a better solution to avoid adding extra complexity.

Reply:
If you use a String like that then the `KroxyliciousExtension` won't play nicely with other parameter resolving extensions like `ValueSource` or `MethodSource`.

This would be better as it means that our extension won't 'claim' String parameters it won't process. It'll prevent a fight with the other parameter resolvers.  Doing this would address my concern.

```java
    @Override
    public boolean supportsParameter(ParameterContext parameterContext, ExtensionContext extensionContext) throws ParameterResolutionException {
        return parameterContext.getParameter().getType().isAssignableFrom(String.class) && parameterContext.getParameter().getName().toLowerCase().contains("namespace");
    }
```

> This Namespace is not just a simple String that is what we need for our test cases. 

(Non blocking) It is easy to get a namespace name from a namespace object.  You could argue that the a namespace parameter is more self documenting than a String. It'd be trivial to have createNamespaceWithWait return a Namespace, so having a having the extension use that would be easy.





Having 

If you changed